### PR TITLE
[usdShade] Doc change only- Apply MaterialBindingAPI in Bind() example

### PR DESCRIPTION
**Pull request by asluk**
See https://groups.google.com/g/usd-interest/c/auBpX7xCQSQ/m/iUeFlJrSAQAJ  

- [x] I have verified that all unit tests pass with the proposed changes

- [X] I have submitted a signed Contributor License Agreement

### CODE REVIEW

**Comment by tallytalwar** [comment]
Oops nice find. I thought I covered it extensively :D.

File: pxr/usd/usdShade/materialBindingAPI.h [hunk 1]
```diff
@@ -917,8 +917,8 @@ class UsdShadeMaterialBindingAPI : public UsdAPISchemaBase
     ///                 "metalSubset", metalFaces);
     /// 
     /// // Bind materials to the created geom-subsets.               
-    /// UsdShadeMaterialBindingAPI(pasticSubset.GetPrim()).Bind(plastic)
-    /// UsdShadeMaterialBindingAPI(metalSubset.GetPrim()).Bind(metal)
+    /// UsdShadeMaterialBindingAPI::Apply(pasticSubset.GetPrim()).Bind(plastic)
```
Comment (tallytalwar) [inline pxr/usd/usdShade/materialBindingAPI.h]: While you are here @asluk, can you also fix the typo here? s/pasticSubset/plasticSubset/

Comment (tallytalwar) [reply pxr/usd/usdShade/materialBindingAPI.h]: Also on line 913 above, there is no concept as UsdGeomMaterialBindingAPI, it should be UsdShadeMaterialBindingAPI, probably also a typo.

Comment (asluk) [reply pxr/usd/usdShade/materialBindingAPI.h]: done-- thanks!

**Comment by jesschimein** [comment]
Filed as internal issue #USD-8817

**Pull request merged**